lib/modules/sourceSnudown.js

```diff
@@ -40,6 +40,7 @@ function attachViewSourceButtons(entry) {
 		}
 
 		$menus::forEachChunked(item => {
+			if (item.querySelector('li.viewSource')) return; // guard against adding "source" button multiple times for whatever buggy reason
```

Comment:
`item` needs to be `$(item).closest('.flat-list')` or something because it's the comments link, not the entire menu